src/controller/controldevice/controller/mapping/factories/ButtonMappingFactory.cpp

```diff
@@ -147,74 +153,74 @@ ButtonMappingFactory::CreateDefaultSDLButtonMappings(ShipDeviceIndex shipDeviceI
-                                                                              SDL_CONTROLLER_BUTTON_LEFTSHOULDER));
+                                                                              LUS_DEFAULT_CONT_MAPPING_L));
             }
             break;
         case BTN_R:
             mappings.push_back(std::make_shared<SDLAxisDirectionToButtonMapping>(shipDeviceIndex, portIndex, BTN_R,
-                                                                                 SDL_CONTROLLER_AXIS_TRIGGERRIGHT, 1));
+                                                                                 LUS_DEFAULT_CONT_MAPPING_R, 1));
             break;
         case BTN_Z:
             mappings.push_back(std::make_shared<SDLAxisDirectionToButtonMapping>(shipDeviceIndex, portIndex, BTN_Z,
-                                                                                 SDL_CONTROLLER_AXIS_TRIGGERLEFT, 1));
+                                                                                 LUS_DEFAULT_CONT_MAPPING_Z, 1));
             break;
         case BTN_START:
             mappings.push_back(std::make_shared<SDLButtonToButtonMapping>(shipDeviceIndex, portIndex, BTN_START,
-                                                                          SDL_CONTROLLER_BUTTON_START));
+                                                                          LUS_DEFAULT_CONT_MAPPING_START));
             break;
         case BTN_CUP:
             mappings.push_back(std::make_shared<SDLAxisDirectionToButtonMapping>(shipDeviceIndex, portIndex, BTN_CUP,
-                                                                                 SDL_CONTROLLER_AXIS_RIGHTY, -1));
+                                                                                 LUS_DEFAULT_CONT_MAPPING_CUP, -1));
             break;
         case BTN_CDOWN:
             mappings.push_back(std::make_shared<SDLAxisDirectionToButtonMapping>(shipDeviceIndex, portIndex, BTN_CDOWN,
-                                                                                 SDL_CONTROLLER_AXIS_RIGHTY, 1));
+                                                                                 LUS_DEFAULT_CONT_MAPPING_CDOWN, 1));
             if (isGameCube) {
                 mappings.push_back(std::make_shared<SDLButtonToButtonMapping>(shipDeviceIndex, portIndex, BTN_CDOWN,
-                                                                              SDL_CONTROLLER_BUTTON_RIGHTSHOULDER));
+                                                                              LUS_DEFAULT_CONT_MAPPING_CDOWN2));
             }
             break;
         case BTN_CLEFT:
             mappings.push_back(std::make_shared<SDLAxisDirectionToButtonMapping>(shipDeviceIndex, portIndex, BTN_CLEFT,
-                                                                                 SDL_CONTROLLER_AXIS_RIGHTX, -1));
+                                                                                 LUS_DEFAULT_CONT_MAPPING_CLEFT, -1));
```

Comment:
the `-1` is important here as it is what determines that we mean left on the x axis instead of right. `LUS_DEFAULT_CONT_MAPPING_CLEFT` is only the axis part of `AxisDirection`

Reply:
I was copying the name of BTN_CLEFT

Reply:
you replaced `SDL_CONTROLLER_AXIS_RIGHTX` with `LUS_DEFAULT_CONT_MAPPING_CLEFT`

`SDL_CONTROLLER_AXIS_RIGHTX` is used for both left ***and*** right